Makefile

```diff
@@ -67,6 +67,9 @@ ios:
 test: all
 	cd client; build/env.sh go run build/ci.go test
 
+test_network: all
+	cd client/knode/genesis && go test . || curl -X POST -H 'Content-type: application/json' --data '{"attachments":[{"title":"Build failure","pretext":"Some days, you just cant get rid of a bomb!","text":"*Network config has changed!* Regenerate golden files, this will also require a network restart.","mrkdwn_in":["text","pretext"],"color":"#ff0000"}]}' https://hooks.slack.com/services/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
```

Comment:
isn't that a private URL? I feel this URL, or pieces of it, shouldn't be here. We should use a secret from drone and use it here as an environment variable.